minio/api.py

```diff
@@ -336,7 +336,8 @@ def get_bucket_policy(self, bucket_name, prefix=""):
 
         return policy.get_policy(statements, bucket_name, prefix)
 
-    def set_bucket_policy(self, policy_access, bucket_name, prefix=""):
+    def set_bucket_policy(self, bucket_name, prefix="",
+                          policy_access=policy.Policy.NONE):
```

Comment:
Why do we need remove as default? In most of the cases a valid bucket policy is set.  Having `Policy.NONE` as default would lead to loosing the policy when the API is called with wrong code.  I feel making `policy_access` mandate to avoid such errors.